### ISSUE TITLE
Separate encoding

### ISSUE THREAD

**Issue by ghost** [DeBukkIt/SimpleFileStorage#11]
What do you think about making separate encoding. It would be good because it will make it platform(and java ) independent.

**Comment by DeBukkIt** [comment]
Are you talking about mechanisms like JSON, GSON or XML?

**Comment by ghost** [comment]
No, i am talking about binary encoding. Like using protobufs or our encoding system.
JSON and XML are too slow.
